packages/forms/src/widgets/EnumerationWidget/EnumerationWidget.js

```diff
@@ -70,6 +70,7 @@ class EnumerationForm extends React.Component {
 		this.timerSearch = null;
 		this.allowDuplicate = false;
 		this.allowImport = false;
+		this.disabledAction = props.schema ? props.schema.disabled : false;
```

Comment:
No need to store it in the instance. You only use it in constructor. Use a const.
```
const 
disabled = props.schema && props.schema.disabled;
```